### PR TITLE
Adds pagination to Posts

### PR DESCRIPTION
**Pull request by Illizian**

### CODE REVIEW

File: src/pages/tag/[tagSlug].tsx [hunk 1]
```diff
@@ -14,14 +15,40 @@ export async function getStaticPaths() {
...
+  //   })
+  //   .flat();
+
   return {
-    paths: getStaticTagPaths(),
+    paths: [
+      { params: { tagSlug: 'Article', page: '2' } },
+      { params: { tagSlug: 'NorDevCon', page: '2' } },
+      { params: { tagSlug: 'Announcement', page: '2' } }
+    ],
```
Comment (Illizian) [inline src/pages/tag/[tagSlug].tsx:43]: :sob:
```
> Build error occurred
Error: A required parameter (page) was not provided as a string in getStaticPaths for /tag/[tagSlug]/[page]
```

Comment (Illizian) [reply src/pages/tag/[tagSlug].tsx:43]: PEBCAK :man_facepalming:

File: src/pages/posts/p/[page].tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  };
+}
+
+type Props = {
+  posts: PostData[];
+  page: number;
+  total: number;
+};
+
+export default function WordsRoute({ posts, page, total }: Props) {
```
Comment (shaunchurch) [inline src/pages/posts/p/[page].tsx:40]: PageRoute?

**Review by shaunchurch — APPROVED** [review]

File: src/pages/posts/p/[page].tsx (hunk 2 above)
Comment (Illizian) [reply src/pages/posts/p/[page].tsx:40]: Yeah, I'm gonna refactor a bunch of these export names along with the relative paths. I think the Posts export is the same.

**Pull request merged**